### ISSUE TITLE
Add an Append to Clipboard button

### ISSUE THREAD

**Issue by Mike-Reddick** [CanisLupus/swift-selection-search#167]
Please Add an Append to Clipboard button, like the "copy" button but to append.

**Comment by CanisLupus** [comment]
Hello and thanks for the suggestion! I'll consider this. The "append to clipboard" is something that I think was asked previously by someone, but the "append to text box" was definitely not.

When appending to the clipboard, this would simply append to the text previously copied in SSS (either via copy or append) and not to the text currently on the clipboard (for example from other apps), otherwise SSS will need a new permission that some users will find scary (read from clipboard).

**Comment by Mike-Reddick** [comment]
I think which just one unified clipboard can be better to the user, to avoid some confusion because of "two clipboards", I have no problem with this new permission, but I understand you, if you think this can be an issue, so implement this separated clipboard. Maybe possible an setting to choose the behavior? I don't know if firefox support permission by demand?! Thank you !

**Comment by CanisLupus** [comment]
Yes, asking for a permission only when it is needed for the first time is possible in Firefox. In that case it would have to be requested when the user enables the append button(s) somewhere in the options page. I guess this will have to be seen in more detail later. :)

**Comment by Mike-Reddick** [comment]
> it would have to be requested when the user enables the append button(s) somewhere in the options page

Perfect!
